src/ui/public/accessibility/angular_aria.js

```diff
@@ -0,0 +1,12 @@
+import 'angular-aria';
+import { uiModules } from 'ui/modules';
+
+uiModules
```

Comment:
I know people can always just RTFM, but I think it's worth documenting the point of this module here in case anyone comes across it. In fact, your PR description is so good I think we can just copy/paste that:

```js
/**
 * This module will take care of attaching appropriate aria tags related to some angular stuff,
 * e.g. it will attach aria-invalid if the model state is set to invalid.
 *
 * You can find more infos in the official documentation: https://docs.angularjs.org/api/ngAria.
 *
 * I just disabled three settings, so it won't automatically attach `tabindex`, `role=button` or
 * handling keyboad events for `ngClick` directives. We have fixes this already in Kibana and using
 * our own directive `kbnAccessibleClick` for this where needed. Enabling this would possibly
 * introduce the attributes in some places where we don't actually want it (e.g. dialog backdrops).
 */
```